core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -114,9 +114,16 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
       consumer.subscribe(topics.toList.asJava, new WrappedAutoPausedListener(consumer, listener))
     case SubscribePattern(pattern, listener) =>
       consumer.subscribe(Pattern.compile(pattern), new WrappedAutoPausedListener(consumer, listener))
-    case Poll =>
-      poll()
-      currentPollTask = schedulePollTask()
+    case Poll(target) =>
+      if (target == self) {
```

Comment:
no, it must be an instance check, the `ActorRef` will be the same after restart.
`target: KafkaConsumerActor` and `target == this`